### PR TITLE
Fix fonts in build status page

### PR DESCRIPTION
**Pull request by mfocko**
Signed-off-by: Matej Focko <mfocko@redhat.com>

### CODE REVIEW

File: packit_service/service/static/style.css [hunk 1]
```diff
@@ -2,7 +2,9 @@
   font-family: sans-serif;
 }
 
-pre {
+pre,
+code,
+kbd {
```
Comment (csomh) [inline packit_service/service/static/style.css:7]: Wow, HTML really turned fancy :smile:  

How about adding `samp` and `var`, too? Just in case :)

Comment (mfocko) [reply packit_service/service/static/style.css:7]: I'd be thumbs up for Latin Modern for `var` :smirk: :joy: gonna add them

File: packit_service/service/static/style.css [hunk 2]
```diff
@@ -2,7 +2,11 @@
   font-family: sans-serif;
 }
 
-pre {
+pre,
+code,
+kbd,
+var,
+samp {
```
Comment (TomasTomecek) [inline packit_service/service/static/style.css:9]: what are these? :D

**Review by TomasTomecek — APPROVED** [review]

File: packit_service/service/static/style.css (hunk 2 above)
Comment (TomasTomecek) [reply packit_service/service/static/style.css:9]: answered myself: https://stackoverflow.com/questions/32284477/what-are-the-differences-between-kbd-samp-and-code-in-html

**Review by csomh — APPROVED** [review]

**Pull request merged**